src/scripts/modules/media/footer/attribution/attribution-template.html

```diff
@@ -8,38 +8,38 @@ <h2 class="medium-header" data-l10n-id="textbook-view-textbook-attribute-header"
   <span data-l10n-id="textbook-view-textbook-content-produced" data-l10n-args='{"url":"{{license.url}}","title":"{{license.name}} {{license.version}}","authors":{{toJSON (mapField authors "fullname")}} }'>
     Textbook content produced by
     <span>{{#each authors}}<span class="list-comma">{{fullname}}</span>{{/each}}</span>
-    is licensed under a <a href="{{license.url}}">{{license.name}} {{license.version}}</a> license.
+    is licensed under a <a href="{{license.url}}">{{license.name}} {{license.version}} International License</a>.
```

Comment:
The problem here is that the proper official name of the license (per Creative Commons) is "Attribution 4.0 International". So, appending the version number to the name will display "Creative Commons Attribution 4.0 International 4.0". 

But, I don't think want to edit the template code to remove the `license.version` since it affects the display of other licenses. 

The license name and version live in the database. Thus I think this is something only @reedstrm can address (hurray). He was trying to come up with a solution to this issue.

@reedstrm: what if we hardcoded the version number in the `license.name` in `licenses.sql`, and in the template code removed the appended `license.version`?